File: program_data/ProgramData/80/591.cpp
int main()
{int y1,y2,m1,m2,d1,d2,i,j,t,p,x1,x2,x3;
int e[13]={0,0,31,59,90,120,151,181,212,243,273,304,334};
   scanf("%d%d%d\n%d%d%d",&y1,&m1,&d1,&y2,&m2,&d2);

   if(y1<y2) {t=y1;y1=y2;y2=t;t=m1;m1=m2;m2=t;t=d1;d1=d2;d2=t;}
   for(i=y2,p=0;i<y1;i++)
   {if((i%4==0&&i%100!= 0) || (i%400==0))   p++;
   }
   x1=365*(y1-y2)+p;
    
   t=0;
    if(((y1%4==0&&y1%100!=0) || (y1%400==0))&&m1>2) t=1;
    x2=e[m1]+d1+t;

    t=0;
    if(((y2%4==0&&y2%100!=0) || (y2%400==0))&&m2>2) t=1;
    x3=e[m2]+d2+t;
   
   x1=x1+x2-x3;
   if(x1<0) x1=-x1;
   printf("%d\n",x1);

return 0;
}

File: program_data/JavaProgramData/20/591.java
package <missing>;

public class GlobalMembers
{
	public static void Main()
	{
	   int x = 0;
	   int i;
	   String s1 = new String(new char[100]);
	   String s2 = new String(new char[100]);
	   String s3 = new String(new char[100]);
	   String s4 = new String(new char[100]);
	   while (scanf("%s %s",s1,s2) != EOF)
	   {
			char max = '0';
			int maxi = 0;
			int len1 = s1.length();
		  //  printf("%s\n",s1);
		  //  printf("%d\n",len1);
			for (i = 0;i < len1;i++)
			{
				if (s1.charAt(i) > max)
				{
					max = s1.charAt(i);
					maxi = i;
				}
			}
			s3 = s1;
		   // printf("%s\n",s3);
			s3 = tangible.StringFunctions.changeCharacter(s3, maxi + 1, '\0');
			s3 += s2;
		  //  printf("%s\n",s3);
			s4 = tangible.StringFunctions.changeCharacter(s4, 0, '\0');
			for (i = maxi + 1;i <= len1;i++)
			{
				s4 = tangible.StringFunctions.changeCharacter(s4, i - maxi, s4.charAt(i - maxi - 1));
				s4 = tangible.StringFunctions.changeCharacter(s4, i - maxi - 1, s1.charAt(i));
			}
		  //  printf("%s\n",s4);
			s3 += s4;
			System.out.printf("%s\n",s3);
	   }
	}
}

